nixos/modules/services/databases/couchdb.nix

```diff
@@ -157,12 +157,15 @@ in {
         mkdir -p ${cfg.databaseDir};
         mkdir -p ${cfg.viewIndexDir};
         touch ${cfg.configFile}
+        touch ${cfg.logFile}
 
         if [ "$(id -u)" = 0 ]; then
-          chown ${cfg.user}:${cfg.group} ${cfg.uriFile}
+          chown ${cfg.user}:${cfg.group} `dirname ${cfg.uriFile}`;
+          -f ${cfg.uriFile} && chown ${cfg.user}:${cfg.group} ${cfg.uriFile} || true
```

Comment:
Use parenthesis, like `(-f ${cfg.uriFile} && chown ${cfg.user}:${cfg.group} ${cfg.uriFile}) || true` to avoid any ambiguity please.